code/modules/client/client_preferences_sounds.dm

```diff
@@ -0,0 +1,107 @@
+/// Controls hearing ambience
+/datum/preference/toggle/sound_ambience
+	category = PREFERENCE_CATEGORY_GAME_PREFERENCES
+	savefile_key = "sound_ambience"
+	savefile_identifier = PREFERENCE_PLAYER
+
+/// Controls hearing announcement sounds
+/datum/preference/toggle/sound_announcements
+	category = PREFERENCE_CATEGORY_GAME_PREFERENCES
+	savefile_key = "sound_announcements"
+	savefile_identifier = PREFERENCE_PLAYER
+
+/// Controls hearing the combat mode toggle sound
+/datum/preference/toggle/sound_combatmode
+	category = PREFERENCE_CATEGORY_GAME_PREFERENCES
+	savefile_key = "sound_combatmode"
+	savefile_identifier = PREFERENCE_PLAYER
+
+/// Controls hearing round end sounds
+/datum/preference/toggle/sound_endofround
+	category = PREFERENCE_CATEGORY_GAME_PREFERENCES
+	savefile_key = "sound_endofround"
+	savefile_identifier = PREFERENCE_PLAYER
+
+/// Controls hearing instruments
+/datum/preference/toggle/sound_instruments
```

Comment:
If you're porting all these prefs you need to
1) remove the old prefs
2) make the new prefs be used

Reply:
removing this